habitat-lab/habitat/utils/visualizations/utils.py

```diff
@@ -235,6 +244,11 @@ def observations_to_image(observation: Dict, info: Dict) -> np.ndarray:
             if obs_k.shape[2] == 1:
                 obs_k = np.concatenate([obs_k for _ in range(3)], axis=2)
             render_obs_images.append(obs_k)
+            import matplotlib.pyplot as plt
+            plt.imshow(obs_k)
+            # plt.colorbar()
+            plt.axis('off')
+            plt.savefig('video_dir/image_dir/'+image_name+'.png', bbox_inches='tight', pad_inches=0)
```

Comment:
Can you add an config option `habitat_baselines.eval.save_image` in yaml config to control whether to save images. You can refer to `habitat_baselines.eval.video_option` in yaml config as an example.

Reply:
Solved:)